sdk/javascript/src/nem/models_ts.js

```diff
@@ -0,0 +1,632 @@
+/* eslint-disable max-len */
+
+import { Address } from './Network.js';
+import * as models from './models.js';
+import { Hash256, PublicKey } from '../CryptoTypes.js';
+
+/**
+ * Type safe descriptor used to generate a descriptor map for AccountKeyLinkTransactionV1Descriptor.
+ *
+ * binary layout for an account key link transaction (V1, latest)
+ */
+export class AccountKeyLinkTransactionV1Descriptor {
+	/**
+	 * Creates a descriptor for AccountKeyLinkTransactionV1.
+	 * @param {models.LinkAction} linkAction link action
+	 * @param {PublicKey} remotePublicKey public key of remote account to which importance should be transferred
+	 */
+	constructor(linkAction, remotePublicKey) {
+		this.rawDescriptor = {
+			type: 'account_key_link_transaction_v1',
+			linkAction,
+			remotePublicKey
+		};
+	}
+
+	/**
+	 * Builds a representation of this descriptor that can be passed to a factory function.
+	 * @returns {object} Descriptor that can be passed to a factory function.
+	 */
+	toMap() {
+		return this.rawDescriptor;
+	}
+}
```

Comment:
missing a space?

Reply:
where? 🤔

Reply:
Between the generated classes.  So between the `}` and the start of the comment for the next class.

```
	toMap() {
		return this.rawDescriptor;
	}
}
/**
 * Type safe descriptor used to generate a descriptor map for NamespaceIdDescriptor.
 *
 * binary layout for a namespace id
 */
export class NamespaceIdDescriptor {
```

should be 

```
	toMap() {
		return this.rawDescriptor;
	}
}

/**
 * Type safe descriptor used to generate a descriptor map for NamespaceIdDescriptor.
 *
 * binary layout for a namespace id
 */
export class NamespaceIdDescriptor {
```